common/src/punc.rs

```diff
@@ -3,12 +3,15 @@ pub enum Punc {
   Comma,
   Colon,
   Dot,
+  SemiColon,
```

Comment:
When are semicolons used in whistle? I personally dislike them because they are in most cases not used for any good reason because statement and expression termination is most of the time implied without the use of them.